spvc/test/run_spirv_cross_tests.py

```diff
@@ -45,127 +43,135 @@
 test_count = 0
 pass_count = 0
 
+devnull = None
 not_used, tmpfile = tempfile.mkstemp()
-devnull = open(os.devnull, 'w')
 
-def mkdir_p(filepath):
-    ''' make sure directory exists to contain given file '''
-    dirpath = os.path.dirname(filepath)
-    try:
-        os.makedirs(dirpath)
-    except OSError as exc:
-        if not (exc.errno == errno.EEXIST and os.path.isdir(dirpath)):
-            raise
+# Quietly run a command.  Throw exception on failure.
+def check_call(cmd):
+    global devnull
```

Comment:
What does this do?

Reply:
You mean why the check_call() function?  No strong need for it now (just redirects stdout).  I expect to add logging here.

Reply:
Sorry, the global devnull thing? Haven't seen that before

Reply:
Just a handle to /dev/null which is opened/closed in main().
Yeah globals suck, but it doesn't seem much of an improvement to put this whole file into a class.